src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/ServerSentEventsTransport.cs

```diff
@@ -138,7 +138,11 @@ private async Task ProcessEventStream(HttpResponseMessage response, Cancellation
             static void CancelReader(object? state) => ((PipeReader)state!).CancelPendingRead();
 
             using (response)
+#if NETCOREAPP
+            using (var stream = await response.Content.ReadAsStreamAsync(cancellationToken))
```

Comment:
Remove
a. This token does nothing in the called code
https://github.com/dotnet/runtime/blob/fbf5e79401363a8ce2922be265062ebca07402cd/src/libraries/System.Net.Http/src/System/Net/Http/HttpContent.cs#L572-L576
https://github.com/dotnet/runtime/blob/fbf5e79401363a8ce2922be265062ebca07402cd/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionResponseContent.cs#L77
b. I don't want to reason in the upstream code about this throwing
c. The `#if` makes the code ugly